scripts/Imaging/imgpy/docs/functionality.md

```diff
@@ -0,0 +1,142 @@
+                        Specify the Slice IDs to which the centers of rotation from --cors correspond.
+                        The number of slices passed here MUST be the same as the number of CORs provided.
+                        The slice IDs MUST be ints. If no slice IDs are provided, then only 1 COR is expected and will be used for the whole stack.
+## Verbosity
+  -v VERBOSITY, --verbosity VERBOSITY
+                        0 - Silent, no text output at all, except results (not recommended)
+                        1 - Low verbosity, will output text on step name
+                        2 - Normal verbosity, will output step name and execution time
+                        3 - High verbosity, will output step name, execution time and memory usage before and after each step
+                        Default: 2 - Normal verbosity.
+## Overwrite all conflicting files in directory.
+  -w, --overwrite-all   Overwrite all conflicting files found in the output directory.
+
+Run Modes:
+## Convert images from one file format to another
+  --convert             Convert images to a different format.
+
+## Prefix for saved out files from conversion
+  --convert-prefix CONVERT_PREFIX
+                        Prefix for saved out files from conversion.
+
+## Image operator mode
+  --imopr [IMOPR [IMOPR ...]]
+                        Image operator currently supports the following operators: ['recon', 'sino', 'show', 'vis', 'cor', 'corvo', 'corpc', 'corwrite', 'sum', '+', 'subtract', 'sub', '-', 'divide', 'div', '/', 'multiply', 'mul', '*', 'mean', 'avg', 'x']
+
+## Aggregate energy levels
+  --aggregate [AGGREGATE [AGGREGATE ...]]
+                        Aggregate the selected image energy levels. The expected input is --aggregate <start> <end> <method:{sum, avg}>... to select indices.
+                                          There must always be an even lenght of indices: --aggregate 0 100 101 201 300 400 sum
+## Aggregate angles
+  --aggregate-angles [AGGREGATE_ANGLES [AGGREGATE_ANGLES ...]]
+                        Select which angles to be aggregated with --aggregate.
+                        This can be used to spread out the load on multiple nodes.
+                        Sample command: --aggregate-angles 0 10, will select only angles 0 - 10 inclusive.
+
+## Aggregate single folder output
+  --aggregate-single-folder-output
+                        The output will be images with increasing number in a single folder.
+
+## Debug, tries to connect to a remote debugger at the specified port with pydevd
+  --debug               Run debug to specified port, if no port is specified, it will default to 59003
+
+## Port on which to look for a debugger
+  --debug-port DEBUG_PORT
+                        Port on which a debugger is listening.
+
+# Reconstruction options:
+## Tool to be used for reconstruction
+  -t {tomopy,astra}, --tool {tomopy,astra}
+                        Tomographic reconstruction tool to use.
```

Comment:
I've added the defaults, about the pros and cons, I'll add a TODO in the descriptions, but for now I am not sure